charts/kubeai/values-eks.yaml

```diff
@@ -0,0 +1,29 @@
+resourceProfiles:
+  nvidia-gpu-l4:
+    nodeSelector:
+      # This only works if you use Karpenter.
+      karpenter.k8s.aws/instance-gpu-name: "l4"
+  nvidia-gpu-l40s:
+    nodeSelector:
+      # This only works if you use Karpenter.
+      karpenter.k8s.aws/instance-gpu-name: "l40s"
+  nvidia-gpu-h100:
+    nodeSelector:
+      # H100 have 8 GPU machine shapes only. This doesn't require Karpenter.
+      node.kubernetes.io/instance-type: "p5.48xlarge"
+  nvidia-gpu-a100-80gb:
+    nodeSelector:
+      # A100 have 8 GPU machine shapes only. This doesn't require Karpenter.
+      node.kubernetes.io/instance-type: "p4d.24xlarge"
+  nvidia-gpu-a100-40gb:
+    nodeSelector:
+      # A100 have 8 GPU machine shapes only. This doesn't require Karpenter.
+      node.kubernetes.io/instance-type: "p4de.24xlarge"
+
+cacheProfiles:
+  efs-dynamic:
+    sharedFilesystem:
+      storageClassName: "efs-sc"
```

Comment:
Maybe leave these off until the EFS guide is added?

Reply:
I think it's good to show these as an example already. It doesn't hurt to have these irrelevant of the EFS guide. Will keep for now.